packages/pds/src/db/migrations/20221021T162202001Z-init.ts

```diff
@@ -43,23 +43,11 @@ export async function up(db: Kysely<unknown>, dialect: Dialect): Promise<void> {
   await db.schema
     .createTable(userTable)
     .addColumn('username', 'varchar', (col) => col.primaryKey())
-    .addColumn('email', 'varchar', (col) => col.notNull())
+    .addColumn('email', 'varchar', (col) => col.notNull().unique())
     .addColumn('password', 'varchar', (col) => col.notNull())
     .addColumn('lastSeenNotifs', 'varchar', (col) => col.notNull())
     .addColumn('createdAt', 'varchar', (col) => col.notNull())
     .execute()
-  await db.schema
```

Comment:
im good to delete these right? or are they serving another purpose?

Reply:
To be honest part of me likes enforcing uniqueness this way, but I agree that these are redundant with using application code to enforce usernames and emails be stored in lowercase.  @pfrazee do you see any issue with "forgetting" the casing of usernames?

Reply:
im not opposed to leaving them in as an extra sanity check

Reply:
I'm not too hung-up on it, either way is cool!